test/integration/programs/sidebar/action-sidebar.js

```diff
@@ -558,6 +558,8 @@ context('program action sidebar', function() {
       .next()
       .should('contain', 'E Form')
       .next()
+      .should('contain', 'Submit Hidden Form')
```

Comment:
Let's add `fx.data = _.first(fx.data, 6)` to the `routeForms` rather than update this with new form names

Reply:
I just updated to that but with a caveat that I commented on below.